### PR TITLE
fix: fix a11y in emptystate

### PR DESCRIPTION
**Pull request by leleOFA**
## Short description
fix a11y in emptystate
## List of changes proposed in this pull request
- insert tab index and arialabel in typography

## How to test
using voiceover see you empty state table

### CODE REVIEW

File: packages/pn-commons/src/components/EmptyState.tsx [hunk 1]
```diff
@@ -74,6 +80,9 @@ function EmptyState({
             <Typography
               color="primary"
               variant="body2"
+              component="p"
```
Comment (AndreaCimini90) [inline packages/pn-commons/src/components/EmptyState.tsx:83]: component="p" is useless

File: packages/pn-commons/src/components/EmptyState.tsx [hunk 2]
```diff
@@ -85,7 +94,13 @@ function EmptyState({
       {secondaryMessage.emptyMessage && (
         <>
           &nbsp;
-          <Typography variant="body2" sx={{ display: 'inline' }}>
+          <Typography
+            variant="body2"
+            component="p"
```
Comment (AndreaCimini90) [inline packages/pn-commons/src/components/EmptyState.tsx:99]: as before

File: packages/pn-commons/src/components/EmptyState.tsx [hunk 3]
```diff
@@ -97,6 +112,9 @@ function EmptyState({
             color="primary"
             variant="body2"
             fontWeight={'bold'}
+            component="p"
```
Comment (AndreaCimini90) [inline packages/pn-commons/src/components/EmptyState.tsx:115]: as before

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-commons/src/components/EmptyState.tsx (hunk 1 above)
Comment (leleOFA) [reply packages/pn-commons/src/components/EmptyState.tsx:83]: delete it

File: packages/pn-commons/src/components/EmptyState.tsx (hunk 2 above)
Comment (leleOFA) [reply packages/pn-commons/src/components/EmptyState.tsx:99]: delete it

File: packages/pn-commons/src/components/EmptyState.tsx (hunk 3 above)
Comment (leleOFA) [reply packages/pn-commons/src/components/EmptyState.tsx]: delete it

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**